index.bs

```diff
@@ -2088,22 +2142,28 @@ Methods</h4>
 			3. In case of failure, queue a task on the <a>control thread</a> to execute the following,
 				and abort these steps:
 
-				1. Reject all promises from <a>pendingResumePromises</a> in order, then clear <a>pendingResumePromises</a>.
+				1. Reject all promises from {{AudioContext/[[pending resume promises]]}}
+					in order, then clear {{AudioContext/[[pending resume promises]]}}.
 
 				2. Reject <em>promise</em>.
 
 			4. Queue a task on the <a>control thread</a>'s event loop, to
 				execute these steps:
 
-				1. Resolve all promises from <a>pendingResumePromises</a> in order, then clear <a>pendingResumePromises</a>.
+				1. Resolve all promises from {{AudioContext/[[pending resume
```

Comment:
This seems inconsistent with how we handle `AudioContext`.  There, we clear `[[pending resume promises]]`.  Here we resolve them all in order.